Sources/ComposableArchitecture/CasePaths/EnumReflection.swift

```diff
@@ -43,7 +43,7 @@ extension CasePath where Value == Void {
 /// - Parameters:
 ///   - embed: An enum case initializer.
 ///   - root: A root enum value.
-/// - Returns: Values iff they can be extracted from the given enum case initializer and root enum, otherwise `nil`.
```

Comment:
This one is actually okay. We meant it as "if and only if," which even the standard library uses occasionally! 😄

https://github.com/apple/swift/blob/9cdfb0edfcae6fa0baadee5df3690597138b5d5e/stdlib/public/core/ContiguousArrayBuffer.swift#L504

Reply:
Oh, TIL! 😄